server/conf/env-var-docs.json

```diff
@@ -0,0 +1,475 @@
+        "description": "The database URL.",
+        "type": "string"
+      },
+      "DB_USERNAME": {
+        "description": "The username used to connect to the database.",
+        "type": "string"
+      },
+      "DB_PASSWORD": {
+        "description": "The password used to connect to the database.",
+        "type": "string"
+      }
+    }
+  },
+  "AWS_REGION": {
+    "description": "Region where the AWS SES service exists. If STORAGE_SERVICE_NAME is set to 'aws', it is also the region where the AWS s3 service exists.",
+    "type": "string"
+  },
+  "Email addresses": {
+    "group_description": "Configuration options for [CiviForm email usage](https://docs.civiform.us/it-manual/sre-playbook/email-configuration).",
+    "members": {
+      "SUPPORT_EMAIL_ADDRESS": {
+        "description": "This email address is listed in the footer for applicants to contact support.",
+        "type": "string"
+      },
+      "IT_EMAIL_ADDRESS": {
+        "description": "This email address receives error notifications from CiviForm when things break.",
+        "type": "string"
+      },
+      "STAGING_ADMIN_LIST": {
+        "description": "If this is a staging deployment, the application notification email is sent to this email address instead of the program administrator's email address.",
+        "type": "string"
+      },
+      "STAGING_TI_LIST": {
+        "description": "If this is a staging deployment, the application notification email is sent to this email address instead of the trusted intermediary's email address.",
+        "type": "string"
+      },
+      "STAGING_APPLICANT_LIST": {
+        "description": "If this is a staging deployment, the application notification email is sent to this email address instead of the applicant's email address.",
+        "type": "string"
+      },
+      "AWS_SES_SENDER": {
+        "description": "The email address used for the 'from' email header for emails sent by CiviForm.",
+        "type": "string"
+      }
+    }
+  },
+  "Application File Upload Storage": {
+    "group_description": "Configuration options for the application file upload storage provider",
+    "members": {
+      "STORAGE_SERVICE_NAME": {
+        "description": "What static file storage provider to use.",
+        "type": "string",
+        "values": ["s3", "azure-blob"]
+      },
+      "AWS_S3_BUCKET_NAME": {
+        "description": "s3 bucket to store files in.",
+        "type": "string"
+      },
+      "AWS_S3_FILE_LIMIT_MB": {
+        "description": "The max size (in Mb) of files uploaded to s3.",
+        "type": "string"
+      },
+      "AZURE_STORAGE_ACCOUNT_NAME": {
+        "description": "The azure account name where the blob storage service exists.",
+        "type": "string"
+      },
+      "AZURE_STORAGE_ACCOUNT_CONTAINER": {
+        "description": "Azure blob storage container name to store files in.",
+        "type": "string"
+      },
+      "AZURE_LOCAL_CONNECTION_STRING": {
```

Comment:
It is difficult to do a top-level cloud provider grouping because the provider is determined per dependency.  For example, our current cloud-deploy-infra azure deployment still uses AWS SES for email sending.